app/models/events/pull_request.rb

```diff
@@ -6,6 +6,7 @@
 #  body       :text
 #  closed_at  :datetime
 #  draft      :boolean          not null
+#  html_url   :string
```

Comment:
what is the html url?

Reply:
the html url is an attr sent by github webhook, this attr will have the url to a pull request

Reply:
and what does `html` stands for?

Reply:
have no idea, we do something like a mirror between the attributes sent by github and our attributes model so the github service is able to storing without parsing all the payload